src/main/webapp/app/course/learning-paths/participate/learning-path-graph-sidebar.component.ts

```diff
@@ -0,0 +1,49 @@
+import { AfterViewInit, Component, Input, ViewChild } from '@angular/core';
+import interact from 'interactjs';
+import { faChevronLeft, faChevronRight, faGripLinesVertical, faNetworkWired } from '@fortawesome/free-solid-svg-icons';
+import { LearningPathGraphComponent } from 'app/course/learning-paths/learning-path-graph/learning-path-graph.component';
+
+@Component({
+    selector: 'jhi-learning-path-graph-sidebar',
+    styleUrls: ['./learning-path-graph-sidebar.component.scss'],
+    templateUrl: './learning-path-graph-sidebar.component.html',
+})
+export class LearningPathGraphSidebarComponent implements AfterViewInit {
+    @Input() learningPathId: number;
+    collapsed: boolean;
+    // Icons
+    faChevronLeft = faChevronLeft;
+    faChevronRight = faChevronRight;
+    faGripLinesVertical = faGripLinesVertical;
+    faNetworkWired = faNetworkWired;
+
+    @ViewChild(`learningPathGraphComponent`, { static: false })
+    learningPathGraphComponent: LearningPathGraphComponent;
+
+    ngAfterViewInit(): void {
+        // allows the conversation sidebar to be resized towards the right-hand side
+        interact('.expanded-graph')
+            .resizable({
+                edges: { left: false, right: '.draggable-right', bottom: false, top: false },
+                modifiers: [
+                    // Set maximum width of the conversation sidebar
```

Comment:
Since this is the same code as for the conversation sidebar, maybe one can move this resize functionality into a common component. But this does not have to be done in this PR.
(Maybe only fix the comments ;))

Reply:
I'll create a follow-up PR for that and link it here once it's open 👍